### PR TITLE
#6443: BalloonToolbar should not be displayed for multi cell table selection.

### LINKED ISSUE #6443: Table properties balloon not displaying on cell multiselect in Balloon and Balloon Block editors
## 📝 Provide detailed reproduction steps (if any)

Occurs on latest master

1. Open Ballon or Balloon Block editor
2. Select multiple cells ( with some text content )

**In Balloon editor you don't event need to add any text**

### ✔️ Expected result

Table properties balloon shows up, you are able to edit a table

### ❌ Actual result

Editor balloon toolbar shows up, you can edit only content

Sample from Balloon Block editor:
16_table1

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by niegowski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Fix (ui): Table properties balloon should be displayed on cell multi-select in Balloon and Balloon Block editors. Closes #6443.

---

### Additional information

### CODE REVIEW

File: packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.js [hunk 1]
```diff
@@ -356,6 +364,23 @@ function getBalloonPositions( isBackward ) {
 	];
 }
 
+function selectionContainsOnlyMultipleObjects( selection, schema ) {
+	// It doesn't contain multiple objects if there is only one range.
+	if ( selection.rangeCount <= 1 ) {
```
Comment (oleq) [inline packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.js:369]: A `DocumentSelection` in the model will always have at least one range. AFAIR there's no way to force an editor into a range-less selection state. Only virtual `Selection` instances can be range-less.

Comment (oleq) [inline packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.js:369]: A `DocumentSelection` in the model will always have at least one range. AFAIR there's no way to force an editor into a range-less selection state. Only virtual `Selection` instances can be range-less.

File: packages/ckeditor5-ui/tests/toolbar/balloon/balloontoolbar.js [hunk 2]
```diff
@@ -374,6 +375,15 @@ describe( 'BalloonToolbar', () => {
 			sinon.assert.notCalled( balloonAddSpy );
 		} );
 
+		// https://github.com/ckeditor/ckeditor5/issues/6443
+		it( 'should not add the #toolbarView to the #_balloon when the selection contains more than one fully contained objects', () => {
```
Comment (oleq) [inline packages/ckeditor5-ui/tests/toolbar/balloon/balloontoolbar.js:379]:
```suggestion
		it( 'should not add the #toolbarView to the #_balloon when the selection contains more than one fully contained object', () => {
```

File: packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.js (hunk 1 above)
Comment (niegowski) [reply packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.js]: So this should be strict `selection.rangeCount == 1` ?

**Pull request merged**